docs/debug-kernel.md

```diff
@@ -10,37 +10,33 @@ fails due to the kernel.
 To provide a debug log of the kernel boot, Clear Containers supports an option
 to enable kernel boot logs using the configuration file.
 
-1. Enable debug in `configuration.toml`
+1. Enable kernel boot messages
+
+   Set the `enable_debug=` option in the `[proxy.cc]` section to `true` (assumes a standard configuration file path):
```

Comment:
Set the `enable_debut=` option in the `[proxy.cc]` section to `true`, which assumes a standard configuration file path: